mkinitcpio

```diff
@@ -22,6 +22,8 @@ _d_presets=mkinitcpio.d
 # options and runtime data
 _optmoduleroot= _optgenimg=
 _optcompress= _opttargetdir=
+_optosrelease="/usr/lib/os-release"
```

Comment:
This should first use /etc/os-release if that exists, and /usr/lib/os-release otherwise.

Reply:
I forgot it, but fixed now.